bot/exts/moderation/incidents.py

```diff
@@ -97,10 +98,20 @@ async def make_embed(incident: discord.Message, outcome: Signal, actioned_by: di
         colour = Colours.soft_red
         footer = f"Rejected by {actioned_by}"
 
+    reported_timestamp = discord_timestamp(incident.created_at)
+    relative_timestamp = discord_timestamp(incident.created_at, TimestampFormats.RELATIVE)
+    reported_on_msg = f"*Reported {reported_timestamp} ({relative_timestamp}).*"
+
+    # If the description will be too long (>4096 total characters), truncate the incident content
+    if len(incident.content) > (allowed_content_chars := 4096-len(reported_on_msg)-2):  # -2 for the newlines
+        description = incident.content[:allowed_content_chars-3] + f"...\n\n{reported_on_msg}"
+    else:
+        description = incident.content + f"\n\n{reported_on_msg}"
```

Comment:
4096 characters is quite a lot, do you think we should limit it to something smaller, maybe 2000 or even just 1000 characters, so it's less bulky?

Reply:
I went for 4096 simply because that's the Discord maximum. If we'd prefer to lower it then I'm happy to.

Reply:
I guess since it was already like that and there wasn't an issue, it's probably fine. We can always change it later if we need to